docs/stability.rst

```diff
@@ -0,0 +1,197 @@
+      </tr>
+      <tr>
+        <td align='center'><span class="pendingdep"></span></td>
+        <td>Pending deprecation.  Might be deprecated in a future version.</td>
+      </tr>
+      <tr>
+        <td align='center'><span class="deprecated"></span></td>
+        <td>Deprecated.  Might be removed in a future version.</td>
+      </tr>
+    </table>
+
+PlasmaPy's planned and existing subpackages are:
+
+.. raw:: html
+
+    <table border="1" class="docutils stability" align='center'>
+        <tr>
+            <th class="head">
+                Subpackage
+            </th>
+            <th class="head">
+                &nbsp;
+            </th>
+            <th class="head">
+                Comments
+            </th>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.atomic
+            </td>
+            <td align='center'>
+                <span class="dev"></span>
+            </td>
+            <td>
+                This package is being actively developed and expanded,
+                and there may be backward incompatible changes to the API.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.classes
+            </td>
+            <td align='center'>
+                <span class="dev"></span>
+            </td>
+            <td>
+                The plan for PlasmaPy's base classes is being planned in
+                PLEP 7, which is in the process of being written.  The
+                existing functionality is unstable.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.constants
+            </td>
+            <td align='center'>
+                <span class="stable"></span>
+            </td>
+            <td>
+                We do not anticipate that there will be any major backward
+                incompatible changes within the
+                <tt class="docutils literal"><span class="pre">constants</span></tt>
+                subpackage.  However, the values for constants may be updated
+                when improved values become available.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.diagnostics
+            </td>
+            <td align='center'>
+                <span class="dev"></span>
+            </td>
+            <td>
+                This subpackage is in the early stages of development.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.mathematics
+            </td>
+            <td align='center'>
+                <span class="dev"></span>
+            </td>
+            <td>
+                The
+                <tt class="docutils literal"><span class="pre">mathematics</span></tt>
+                subpackage is in the early stages of development.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                plasmapy.parameters
```

Comment:
Why `plasmapy.parameters` instead of `plasmapy.physics`?

Reply:
I'm pretty sure this is just what happens when I git too little sleep!  Thanks for catching this!

Reply:
> git too little sleep

OOOOOOOOOOH